docs/DeveloperGuide.adoc

```diff
@@ -346,21 +391,18 @@ _{More to be added}_
 [appendix]
 == Non Functional Requirements
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-_{More to be added}_
+1. Should work on any mainstream OS that has JDK 11 installed.
+2. Should be usable by user with novice computing experience.
+3. Should be able to respond to user input within 2 seconds.
+4. Should be able to run fullscreen without any UI issues.
+5. Should be able to read easily by users.
```

Comment:
Is this well-defined? What exactly does read easily by users mean? When can you tell that users can read easily?